src/store/actions.ts

```diff
@@ -204,6 +205,7 @@ export default {
       }))
       commit('updateVariantCounts', variantCounts)
     }
+    commit('setIsGridLoading', false)
```

Comment:
This is not the correct place, I think, if the getters no longer match then another call is still loading.